python/dolfinx/plotting.py

```diff
@@ -447,25 +447,33 @@ def plot(object, *args, **kwargs):
 _dolfin_to_vtk_cell = {cpp.mesh.CellType.point: 1, cpp.mesh.CellType.triangle: 69,
                        cpp.mesh.CellType.quadrilateral: 70, cpp.mesh.CellType.tetrahedron: 71,
                        cpp.mesh.CellType.hexahedron: 72}
-# Permutation for Dolfinx DG layout to VTK (triangles)
-_perm_dg = {1: [0, 1, 2], 2: [0, 2, 5, 1, 4, 3], 3: [0, 3, 9, 1, 2, 6, 8, 7, 4, 5],
-            4: [0, 4, 14, 1, 2, 3, 8, 11, 13, 12, 9, 5, 6, 7, 10]}
-_perm_dq = {1: [0, 1, 3, 2], 2: [0, 2, 8, 6, 1, 5, 7, 3, 4], 3: [0, 3, 15, 12, 1, 2, 7, 11, 13, 14, 4, 8, 5, 6, 9, 10]}
+# Permutation for Dolfinx DG layout to VTK
```

Comment:
Don't we normally have this type of information in the C++ layer?

Reply:
These are a new set of permutations, from the DG lexicographical ordering to VTK.

Reply:
They are not required for any other usage than making meshes out of DG dofmaps, so I'd rather keep them locally in this file.